applications/cudaquantum/cudaquantum-ubuntu/cudaquantum-0.8.0-ubuntu.dockerfile

```diff
@@ -0,0 +1,294 @@
+FROM ubuntu:22.04
+
+#CICD metadata
+LABEL org.opencontainers.image.arch=arm
+LABEL org.opencontainers.image.compilation=auto
+LABEL org.opencontainers.image.devmode=false
+LABEL org.opencontainers.image.noscan=true
+
+#Image metadata
+LABEL org.opencontainers.image.name="cudaquantum"
+LABEL org.opencontainers.image.version="1.0.0"
+LABEL org.opencontainers.image.version="10-11-2024"
+LABEL org.opencontainers.image.minversion="0.0.6"
+LABEL org.opencontainers.image.authors="Shusen Liu <shusen.liu@pawsey.org.au>"
+LABEL org.opencontainers.image.vendor="Pawsey Supercomputing Research Centre"
+LABEL org.opencontainers.image.licenses="GNU GPL3.0"
+LABEL org.opencontainers.image.title="Ella cudaquantum with cuQuantum and hpcx"
+LABEL org.opencontainers.image.description="We provide a container image for the Ella project, \
+which includes the cuQuantum library and the HPC-X MPI library. \
+Pip venv: . /opt/cuquantum-source/cuquantum-env/bin/activate. \
+1. Compile C++ with nvq++ from cudaquantum; \
+2. Run Python with nvqpy from cudaquantum; \
+3. Run MPI with OpenMPI from HPC-X; \
+4. Run Qiskit with cuQuantum and Aer from Qiskit;"
+
+ARG PY_VERSION="3.12"
```

Comment:
Eventually, we'll need to check if this recipe needs revisions when using later versions but for now looks good.